pkg/keyring/algo/ed25519.go

```diff
@@ -0,0 +1,54 @@
+package algo
+
+import (
+	"github.com/cosmos/cosmos-sdk/crypto/types"
+	bip39 "github.com/cosmos/go-bip39"
+
+	ed25519 "crypto/ed25519"
+
+	cosmosEd25519 "github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
```

Comment:
I think you should avoid using cosmos for `ed25519` altogether and simply use the crypto package. It doesn't gain us anything since we won't be using it in relation to the Terra transaction signing.